.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,25 @@
+name: markdownlint
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+    - '**.md'
+  pull_request:
+    branches: [ main ]
+    paths:
+    - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: check out code
```

Comment:
nit: all the steps here are self-describing. I do not see any benefits in using `name` here

Reply:
fixed: 766cc22dd17cb65998b8fb8b5d3d8136192e5e65